### DISCUSSION TITLE
Doubt on File Transfer

### DISCUSSION THREAD

**Discussion by lfmm-brz** [ExtremeElectronics/RC2040#28] [General]
Dear Sir.
I've built the PICO RC2040, everything is working fine, except the File Transfer.
When I press the BUT button the LED does not lit  and "Start?:" does not appear on the Teraterm.
I've checked the connections of the switch and LED and they are OK.
I am using the latest RC2040.uf2 from 'main', also used the RC2040.uf2 that is on folder 'test', got the same.
Thank you.

**Comment by ExtremeElectronics** [comment]
Does your RC2040 boot into CPM, and run CPM programs ?
Are you in CPM when you press the button ?
Do you have a diskdefs file in the SD Card ?

**Reply by lfmm-brz** [reply]
Does your RC2040 boot into CPM, and run CPM programs ?
Yes.

Are you in CPM when you press the button ?
No. I am pressing the button after the SCM prompt.

Do you have a diskdefs file in the SD Card ?
Yes.

**Comment by ExtremeElectronics** [comment]
Are you connecting with USB (not FDTI/UART?) 
(sorry for all the questions)

**Reply by lfmm-brz** [reply]
Yes,  I am connecting the Raspberry Pi Pico to the computer via USB.

**Comment by ExtremeElectronics** [comment]
OK, Ive just tried it on a RC2040
I can get the prompt on all "valid" settings of the address jumpers, by pressing the but immediately after the respective prompts. 
Currently I'm using the SIO binaries. 

If you don't get the prompt, then I can only assume that there is a break in the track somewhere on GPIO8 or a faulty button preventing it from going to earth. 

If you have a jumper wire, can you try connecting a jumper from any of the gnd pins of the Pico and touching GPIO8 (pin 11) to see if that triggers the serial mode.

**Reply by lfmm-brz** [reply]
OK, Ive just tried it on a RC2040
I can get the prompt on all "valid" settings of the address jumpers, by pressing the but immediately after the respective prompts.
Currently I'm using the SIO binaries.

A: My jumpers are all open. I don't remember what binaries I am using, they are untouched, I've downloaded and installed on RBPi Pico and on the SD card.


If you don't get the prompt, then I can only assume that there is a break in the track somewhere on GPIO8 or a faulty button preventing it from going to earth.

A: I've tested with the multimeter, it's ok.


If you have a jumper wire, can you try connecting a jumper from any of the gnd pins of the Pico and touching GPIO8 (pin 11) to see if that triggers the serial mode.

A: Also tested this way, no result.

I am offshore, arriving at home I check everything again and revert to you.
Thank very much.
Regards.

**Comment by lfmm-brz** [comment]
Dear Sir,
I rechecked everything again and found (shame on me) a broken track from GPIO22 (pin 29) to GND. Fixed it and everything is working now. Thank very much for your support.
